diesel_infer_schema/src/inference.rs

```diff
@@ -36,16 +42,28 @@ pub fn establish_connection(database_url: &str) -> Result<InferConnection, Box<E
         _ if database_url.starts_with("postgres://") || database_url.starts_with("postgresql://") => {
             establish_real_connection(database_url).map(InferConnection::Pg)
         }
+        #[cfg(feature = "mysql")]
+        _ if database_url.starts_with("mysql://") => {
+            establish_real_connection(database_url).map(InferConnection::Mysql)
+        }
         #[cfg(feature = "sqlite")]
         _ => establish_real_connection(database_url).map(InferConnection::Sqlite),
-        #[cfg(not(feature = "sqlite"))]
+        #[cfg(all(feature = "postgres", not(feature = "sqlite")))]
```

Comment:
We still need one to win if both are enabled. I can probably make the error message more generic and just have it be `not(cfg(feature = "sqlite"))` in the future.